doc/footer.html

```diff
@@ -17,5 +17,111 @@
 </a> $doxygenversion
 </small></address>
 <!--END !GENERATE_TREEVIEW-->
+<script type="text/javascript">
+//<![CDATA[
+/*
+  StringCompare that treats followed numbers as a single one.
+  Example: "OpenCV 9" would come first than "OpenCV 10"
+*/
+function myStringCompare(ele1, ele2) {
+    var str1 = ele1.innerHTML;
+    var str2 = ele2.innerHTML;
+    var value = -1;
+    var smaller = str1.length;
+    if (str2.length == smaller) {
+        value = 0;
+    } else if (str2.length < smaller) {
+        smaller = str2.length;
+        value = 1;
+    }
+    for (var i = 0; i < smaller; i++) {
+        var str1_tmp_char = str1.charAt(i);
+        var str2_tmp_char = str2.charAt(i);
+        if (isNaN(str1_tmp_char) || isNaN(str2_tmp_char)) {
+            if (str1_tmp_char < str2_tmp_char) {
+                return -1;
+            } else if (str1_tmp_char > str2_tmp_char) {
+                return 1;
+            }
+        } else {
+            var int1 = parseInt(str1.substring(i));
+            var int2 = parseInt(str2.substring(i));
+            if (int1 < int2) {
+                return -1;
+            } else if (int1 > int2) {
+                return 1;
+            }
+        }
+    }
+    return value;
+}
+function addButton(label, buttonName) {
+    var b = document.createElement("BUTTON");
+    b.innerHTML = buttonName;
+    b.setAttribute('class', 'toggleable_button label_' + label);
+    b.onclick = function() {
+        $('.toggleable_button').css({
+            border: '2px outset',
+            'border-radius': '4px'
+        });
+        $('.toggleable_button.label_' + label).css({
+            border: '2px inset',
+            'border-radius': '4px'
+        });
+        $('.toggleable_div').css('display', 'none');
+        $('.toggleable_div.label_' + label).css('display', 'block');
+    };
+    b.style.border = '2px outset';
+    b.style.borderRadius = '4px';
+    b.style.margin = '2px';
+    return b;
+}
+function buttonsToAdd($elements, $heading, $type) {
+    if ($elements.length === 0) {
+        $elements = $("" + $type + ":contains(" + $heading.html() + ")").parent().prev("div.newInnerHTML");
+    }
+    var arr = jQuery.makeArray($elements);
+    arr = arr.sort(myStringCompare);
```

Comment:
Tutorial author is responsible for this, I think it is quite easy to put all blocks in same order.
And with forced sorting we will loose possibility to make manually-sorted buttons, like "prev", "next".